tests/jballerina-unit-test/src/test/resources/test-src/workers/sync-send.bal

```diff
@@ -1,3 +1,20 @@
+// Copyright (c) (2019-2022), WSO2 Inc. (http://www.wso2.com).
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
```

Comment:
Shall we change the order of this import? Usually as a best practice, we need to follow the alphabetical order.